sdk/include/reactos/dbgbitmap.h

```diff
@@ -0,0 +1,359 @@
+/*
+ * PROJECT:     ReactOS SDK
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Helper functions to debug RTL_BITMAP
+ * COPYRIGHT:   Copyright 2024 Timo Kreuzer <timo.kreuzer@reactos.org>
+ */
+
+#pragma once
+
+#include <rtltypes.h>
+#include <rtlfuncs.h>
+
+typedef struct _RTL_BITMAP_DBG
+{
+    union
```

Comment:
Is the only purpose of this union for the SAL annotations?

Reply:
No, it's to avoid casts. The wrapper functions pass `&BitMapHeader->BitMap` and the code using the bitmap can still directly access `SizeOfBitMap` / `Buffer`.